src/leetCodeSolutionProvider.ts

```diff
@@ -63,6 +45,27 @@ class LeetCodeSolutionProvider implements Disposable {
         }
     }
 
+    private addMdDefaultHighlight(): void {
+        // The @types typedef of `highlight` is wrong, which should return a string.
+        // tslint:disable-next-line:typedef
+        const highlight = this.markdown.options.highlight as (code: string, lang?: string) => string;
+        this.markdown.options.highlight = (code: string, lang?: string): string => {
+            return highlight(code, lang || this.solution.lang);
```

Comment:
Under what circumstances will the param `lang` be undefined?

Reply:
There are two ways to define a code block:
1. Fence (with lang specification)
> \```lang
> code here
> \```

2. Code block (indented, no lang specification)
> emmm 
> 
>  code here
>
> emmmm

Which exists in the discussions equally frequent.

The `code_block` does not have a language specification, so it will not be highlighted in the markdown-it rule. 

I decided to use the solution language tag to highlight it, so I alter the `code_block` rule, just as what [`MarkdownEngine.addCodeBlockHighlight`](https://github.com/jdneo/vscode-leetcode/pull/224/files#diff-f7ac20b7043db95594bdb6d86f3ef02bR77) do:
1. Since there is no language information in `code_block` rendering rule, so I pass `undefined` to `lang` parameter.
2. Since I have found in some dicussions that there is mixed usage of `fence` and `code_block`, where the author just intended to put some lang-free message in indented `code_block`, so I decided to not highlight the `code_block` if any `fence` is found.